libcxx/utils/ci/run-buildbot

```diff
@@ -204,6 +204,11 @@ function test-armv7m-picolibc() {
 
     step "Generating CMake for compiler-rt"
     flags="--sysroot=${INSTALL_DIR}"
+    # LLVM_ENABLE_PER_TARGET_RUNTIME_DIR=ON means that we produce a file
+    # libclang_rt.builtins.a that will be installed to lib/armv7m-unknown-none-eabi/.
```

Comment:
I like to see the same path as used below.
```suggestion
    # libclang_rt.builtins.a that will be installed to ${INSTALL_DIR}/lib/armv7m-unknown-none-eabi/.
```

Reply:
Done.